### PR TITLE
Add install node doc, update aws and linux installation instructions

### PR DESCRIPTION
**Pull request by darahayes**

### CODE REVIEW

File: setup-guides/aws-setup-guide.md [hunk 1]
```diff
@@ -5,80 +5,84 @@
...
 
-This setup guide will walk you through installing nscale and it's prerequisites on an AWS AMI.
+This setup guide will walk you through installing nscale and its prerequisites on an AWS AMI.
 
 ## Requirements
 
 * An [Amazon AWS account][AWS-signup]
 
-## Instantiating an _nscale-base_ AMI
+# Create a nscale base image
```
Comment (mcdonnelldean) [inline setup-guides/aws-setup-guide.md]: Make ##

File: setup-guides/aws-setup-guide.md [hunk 2]
```diff
@@ -5,80 +5,84 @@
...
-then click "Review and Launch". If we're happy with the details we can then press
-"Launch". We'll be asked to manage key pairs (we can create one if we don't have one),
-after which we can click the "Launch Instances" button.
+ - Copy the key you used to connect to the instance - nscale will use this key for communicating with other instances it starts. You can do it with an scp command like below:
+ ```bash
+ scp -i my-key.pem my-key.pem ubuntu@52.113.18.118:~/
+ ```
 
-## Logging on to the instance
+# Docker
```
Comment (mcdonnelldean) [inline setup-guides/aws-setup-guide.md]: Make ##

**Comment by darahayes** [comment]
@McDonnellDean Thanks for the heads up. Fixing all heading typos now. No need to report them all.

**Comment by mcdonnelldean** [comment]
@darragh-hayes Are we ok to merge this?

**Comment by darahayes** [comment]
I believe so but I've left it open for someone else to review!

**Comment by mcollina** [comment]
LGTM

**Comment by mcdonnelldean** [comment]
Thanks @mcollina Looks good to me too so going to merge. Cheers @darragh-hayes ya big star!

**Pull request merged**